v2.1/admin-ui-hardware-dashboard.md

```diff
@@ -4,32 +4,20 @@ summary: The Hardware dashboard lets you monitor CPU usage, disk throughput, net
 toc: true
 ---
 
-The **Hardware** dashboard lets you monitor CPU usage, disk throughput, network traffic, storage capacity, and memory. To view this dashboard, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui), click **Metrics** on the left-hand navigation bar, and then select **Dashboard** > **Hardware**.
+The **Hardware** dashboard lets you monitor CPU usage, disk throughput, network traffic, storage capacity, and memory. To view this dashboard, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui), click **Metrics** on the left, and then select **Dashboard** > **Hardware**.
 
 The **Hardware** dashboard displays the following time series graphs:
 
-## User CPU Percent
+## CPU Percent
 
-<img src="{{ 'images/v2.1/admin_ui_user_cpu.png' | relative_url }}" alt="CockroachDB Admin UI User CPU Percent graph" style="border:1px solid #eee;max-width:100%" />
+<img src="{{ 'images/v2.1/admin_ui_cpu_percent.png' | relative_url }}" alt="CockroachDB Admin UI CPU Percent graph" style="border:1px solid #eee;max-width:100%" />
 
-- In the node view, the graph shows the amount of CPU resources used by the CockroachDB process for the selected node.
+- In the node view, the graph shows the percentage of CPU in use by the CockroachDB process for the selected node.
 
-- In the cluster view, the graph shows the amount of CPU resources used by the CockroachDB process across all nodes.
+- In the cluster view, the graph shows the percentage of CPU in use by the CockroachDB process across all nodes.
```

Comment:
nit: I would prefer "for each node" to "across all nodes"

Reply:
Just on this one or for each graph (wherever applicable)?